chain/get_ancestors_test.go

```diff
@@ -32,14 +32,15 @@ func requireGrowChain(ctx context.Context, t *testing.T, blockSource *th.TestFet
 	link := requireHeadTipset(t, chainStore)
 
 	signer, ki := types.NewMockSignersAndKeyInfo(1)
-	mockSignerPubKey := ki[0].PublicKey()
+	minerOwner, err := ki[0].Address()
```

Comment:
Name this `minerWorker` for consistency with the same lines below (or change the others to match this).